service/prometheus/scrapeconfig.go

```diff
@@ -100,6 +103,14 @@ func getScrapeConfig(service v1.Service, certificateDirectory string) config.Scr
 				TargetLabel:  NamespaceLabel,
 				Action:       config.RelabelReplace,
 			},
+			// Relabel http endpoints to scrape via http.
+			{
+				SourceLabels: model.LabelNames{PrometheusServiceNameLabel},
+				TargetLabel:  model.SchemeLabel,
+				Regex:        HTTPEndpointRegexp,
+				Replacement:  HttpScheme,
+				Action:       config.RelabelReplace,
+			},
```

Comment:
Do other have https schema label? Is it by default?

Reply:
Yeah, the default scheme is https.